chart/compass/templates/tests/tenant-fetcher/tenant-fetcher-test.yaml

```diff
@@ -15,10 +15,10 @@ spec:
   template:
     metadata:
       annotations:
-        sidecar.istio.io/inject: "true"
         cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
       labels:
         app: {{ .Chart.Name }}-tenant-fetcher-tests-app
+        sidecar.istio.io/inject: "true"
```

Comment:
Giving Atlas a heads-up on this (annotation replaced by label) could be a good idea